components/infobox/wikis/stormgate/infobox_building_custom.lua

```diff
@@ -144,6 +144,16 @@ function CustomBuilding:nameDisplay(args)
 	return factionIcon .. (args.name or self.pagename)
 end
 
+---@param args table
+---@return string?
+function CustomBuilding:subHeaderDisplay(args)
+	if string.find(args.subfaction, '1v1') or string.find(args.subfaction, self.pagename) then return end
```

Comment:
needs nil check
also what to do for unser subfaction?

Reply:
This was the respecting code from Infobox/Unit - changed it in both Infoboxes to check for nil or if unset. 
If there is nothing specified the subheader will not be shown (only if it's a subfaction unit/building, that isn't used in 1v1).